src/relentless/model/potential/angle.py

```diff
@@ -116,6 +124,9 @@ def energy(self, type_, theta):
 
         theta, u, s = self._zeros(theta)
 
+        # Validate theta
+        theta = self._validate_coordinate(theta)
```

Comment:
It isn't necessary to catch the return value, if you don't want. Also, Python recommends against commenting things that are easily understood from the code
```suggestion
        self._validate_coordinate(theta)
```